### PR TITLE
Unit test improvement and creation of the apartment.xml package

### PR DESCRIPTION
**Pull request by av1m**
At first, we wanted to modify a test class that declared a hand and no tests. This was not necessary and could lead to misunderstanding when reading the documentation. see [here](https://github.com/av1m/Apartments/blob/d7a028ec4184fca5bc5d891de3ed6c94da742eef/Doc/overview.adoc#readtwoapartmentstest)

Then we noticed that we have centralized the json functions in `apartment.json` and we thought that it would be wise to do the same for XML, so we create a new package `apartment.xml`

It is important to note that this PR answers two things (we judged that it was useless to make two PR because it is a lot of refactoring) : 

* Improvement of the `ReadApartmentsXMLFormatTest` class as mentioned in the first paragraph
* Migration of the `readapartments` and `toxmlproperties` package into the `apartment.xml` package

**Please note that apart from the `ReadApartmentsXMLFormatTest.java` file, the files have not been modified (except for imports and paths). Thus, the second goal of this PR is not to improve these other files but to improve the organization of packages**

In addition, the documentation (in `.adoc` format) has been updated

### CODE REVIEW

File: src/test/java/io/github/oliviercailloux/y2018/apartments/apartment/xml/ToXmlPropertiesTest.java [hunk 1]
```diff
@@ -1,11 +1,12 @@
-package io.github.oliviercailloux.y2018.apartments.toxmlproperties;
+package io.github.oliviercailloux.y2018.apartments.apartment.xml;
 
 import java.io.File;
 
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 
+import io.github.oliviercailloux.y2018.apartments.apartment.xml.XMLProperties;
```
Comment (MorganeFt) [inline src/test/java/io/github/oliviercailloux/y2018/apartments/apartment/xml/ToXmlPropertiesTest.java]: Organize imports

**Review by MorganeFt — CHANGES_REQUESTED** [review]
Dans un souci de lisibilité de la PR, nous avons décidé de ne pas prendre en compte les remarques de BP format. De plus, lors de cette itération, nous allons implémenter le google-java-format qui résoudra ce genre de problèmes.

De même, les classes toxmlproperties et ReadApartmentsXMLFormat pourraient être optimisée dans une prochaine PR.

File: src/test/java/io/github/oliviercailloux/y2018/apartments/apartment/xml/ToXmlPropertiesTest.java (hunk 1 above)
Comment (av1m) [reply src/test/java/io/github/oliviercailloux/y2018/apartments/apartment/xml/ToXmlPropertiesTest.java]: Resolve in 599c690cd2e0d2b0bceee3a241d29bd9989a7674

**Comment by av1m** [comment]
Les imports seront également ordonnés à l'aide de google-java-format.

Les imports inutilisés ont été supprimés.

**Review by MorganeFt — APPROVED** [review]

**Review by oliviercailloux — APPROVED** [review]

**Pull request merged**